src/__tests__/lib/containers/drug_upload_tool/UserFileGrid.test.tsx

```diff
@@ -38,16 +34,17 @@ describe('basic tests', () => {
 
   beforeEach(() => {
     SynapseClient.listFormData = jest.fn(() =>
-      Promise.resolve({ page: [mockFileEntity, mockFileEntity] }),
+      Promise.resolve(formListData),
     )
   })
 
   it('displays table with file list when files are present', async () => {
     const { wrapper, instance } = await createShallowComponent(props)
     await instance.componentDidMount()
     expect(wrapper).toBeDefined()
-    expect(wrapper.find('table')).toHaveLength(1)
-    expect(wrapper.find('tbody > tr')).toHaveLength(2)
+    expect(wrapper.find('table')).toHaveLength(2)
```

Comment:
There are now two tables one with submissions in process and another with  already submissions complete